lib/AST/Bridging/DeclBridging.cpp

```diff
@@ -90,6 +90,25 @@ BridgedDeclNameLoc BridgedDeclNameLoc_createParsed(BridgedASTContext cContext,
 #define ABSTRACT_DECL(Id, Parent) DECL(Id, Parent)
 #include "swift/AST/DeclNodes.def"
 
+// Define `.asValueDecl` on each BridgedXXXDecl type that's also a
```

Comment:
What is this for? Doing this kind of unchecked cast (for all kind of decls) looks dangerous.

Reply:
I see, thanks for the explanation.
However, you don't need this because the Decl classes in `Declarations.swift` form the same class hierarchy as on the C++ side. So all decl classes which are value decls are derived from `ValueDecl` and therefore are ValueDecls.
Therefore you can add `setAccess` to `class ValueDecl`.

Reply:
> Therefore you can add setAccess to class ValueDecl

Done, thanks